ngrinder-controller/src/main/resources/messages_cn.properties

```diff
@@ -314,6 +314,7 @@ script.editor.title=\u811a\u672c\u7f16\u8f91\u5668
 script.editor.button.validate=\u9a8c\u8bc1\u811a\u672c
 script.editor.message.overWriteNewer=\u5728\u7cfb\u7edf\u4e2d\u5b58\u5728\u6700\u8fd1\u4fee\u6539\u7684\u811a\u672c\uff0c\u60a8\u8fd8\u8981\u91cd\u5199\u5f53\u524d\u811a\u672c\u5417\uff1f
 script.editor.error.validate=\u9a8c\u8bc1\u811a\u672c\u53d1\u751f\u9519\u8bef\u3002
+script.editor.sample.message=If you want to know how to writing script, click <b>Sample</b> button.<br/>Then you can see the sample script.
```

Comment:
@GwonGisoo That's all right.
script.editor.sample.message=\u70b9\u51fb<b>Sample</b> \u6309\u94ae,\u67e5\u770b\u66f4\u591a\u811a\u672c\u793a\u4f8b